bot/exts/utils/snekbox/_cog.py

```diff
@@ -373,11 +382,39 @@ async def send_job(self, ctx: Context, job: EvalJob) -> Message:
 
                 msg += f"\n{Emojis.failed_file} {blocked_msg}"
 
+            # Split text files
+            text_files = [f for f in allowed if f.suffix in TXT_LIKE_FILES]
+            # Inline until budget, then upload to paste service
+            budget_lines = MAX_OUTPUT_BLOCK_LINES
+            budget_chars = MAX_OUTPUT_BLOCK_CHARS
+            for file in text_files:
+                file_text = file.content.decode("utf-8", errors="replace") or "[Empty]"
+                # Override to always allow 1 line and < 50 chars, since this is less than a link
+                if len(file_text) < 50 and file_text.count("\n") < 2:
```

Comment:
Doesn't one `\n` mean there are two lines?

Reply:
Yup. Fixed in [a6726c5](https://github.com/python-discord/bot/pull/2326/commits/a6726c5ba3e890cc7ccb06dfb25fb61968353e90)